stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -1586,7 +1586,7 @@ Base.reducedim_initarray0(A::SparseMatrixCSC, region, v0, ::Type{R}) where {R} =
     fill!(similar(dims->Array{R}(uninitialized, dims), Base.reduced_indices0(A,region)), v0)
 
 # General mapreduce
-function _mapreducezeros(f, op, ::Type{T}, nzeros::Int, v0) where T
+function _mapreducezeros(f, op, ::Type{T}, nzeros::U, v0) where {T, U<:Integer}
```

Comment:
Perhaps `nzeros::U, v0) where {T, U<:Integer}` -> `nzeros::Integer, v0) where T`?

Reply:
I was under the impression that approach would not generate efficient code because of the abstract type but I just checked the docs and found that my understanding was wrong! Thanks @Sacha0! I'll update the PR with this.